step-templates/sitecore-ship-publish.json

```diff
@@ -0,0 +1,114 @@
+      "HelpText": "Url for the site you are installing the .update package on (ex: _http://mysite.dev_)",
+      "DefaultValue": "",
+      "DisplaySettings": {
+        "Octopus.ControlType": "SingleLineText"
+      },
+      "Links": {}
+    },
+    {
+      "Id": "ff09a832-5736-400d-a438-aa19023138c6",
+      "Name": "PublishTargets",
+      "Label": "Publish Targets",
+      "HelpText": "The db targets for the publish, comma delimited (ex: _preview, web_)",
+      "DefaultValue": "web",
+      "DisplaySettings": {
+        "Octopus.ControlType": "SingleLineText"
+      },
+      "Links": {}
+    },
+    {
+      "Id": "8cdd686d-e01a-45c5-b372-47fda3e55e56",
+      "Name": "PublishLanguages",
+      "Label": "Publish Languages",
+      "HelpText": "The language targets for the publish, comma delimited (ex: _en_)",
+      "DefaultValue": "en",
+      "DisplaySettings": {
+        "Octopus.ControlType": "SingleLineText"
+      },
+      "Links": {}
+    },
+    {
+      "Id": "68396c7e-ccc1-4782-bd0b-5ccdc314478c",
+      "Name": "PublishMode",
+      "Label": "Publish Mode",
+      "HelpText": "The publish mode (ex: smart, full, incremental or listofitems)",
+      "DefaultValue": "smart",
+      "DisplaySettings": {
+        "Octopus.ControlType": "Select",
+        "Octopus.SelectOptions": "smart|smart\nfull|full\nincremental|incremental\nlistofitems|listofitems"
+      },
+      "Links": {}
+    },
+    {
+      "Id": "74027445-6d9c-4f61-81c4-3e170dc15833",
+      "Name": "ConnectionTimeOutInSeconds",
+      "Label": "Connection Timeout",
+      "HelpText": "Connection Timeout in seconds (ex: 300",
+      "DefaultValue": "300",
+      "DisplaySettings": {
+        "Octopus.ControlType": "SingleLineText"
+      },
+      "Links": {}
+    },
+    {
+      "Id": "573a0b9b-c951-4f43-9c58-7263111a51da",
+      "Name": "Username",
+      "Label": "Username",
+      "HelpText": "If your Ship endpoint requires authentication, insert Username here.",
+      "DefaultValue": "",
+      "DisplaySettings": {
+        "Octopus.ControlType": "SingleLineText"
+      },
+      "Links": {}
+    },
+    {
+      "Id": "523ada58-f5b7-4bb4-ab87-12193c58bed7",
+      "Name": "Password",
+      "Label": "Password",
+      "HelpText": "If your Ship endpoint requires authentication, insert Password here.",
+      "DefaultValue": "",
+      "DisplaySettings": {
+        "Octopus.ControlType": "Sensitive"
+      },
+      "Links": {}
+    },
+    {
+      "Id": "b981883e-0b08-4d08-9787-a466eaebaeae",
+      "Name": "PreviousDeployStep",
+      "Label": "Previous Deploy Step",
+      "HelpText": "Use this only if you are publishing the site using the _ListOfItems_ publish mode, associating the previous deploy step will grab the created list",
+      "DefaultValue": "",
+      "DisplaySettings": {
+        "Octopus.ControlType": "StepName"
+      },
+      "Links": {}
+    }
+  ],
+  "Category": "sitecore",
+  "LastModifiedBy" : "svandenbush@geekhive.com",
```

Comment:
Same here about updating this to your :octocat: username, it will be linked correctly on the Library site